packages/session-replay-browser/src/remote-config-fetch.ts

```diff
@@ -1,45 +1,60 @@
 import { BaseTransport } from '@amplitude/analytics-core';
 import { Status } from '@amplitude/analytics-types';
 import { TargetingFlag } from '@amplitude/targeting';
-import { SESSION_REPLAY_SERVER_URL } from './constants';
 import { UNEXPECTED_ERROR_MESSAGE } from './messages';
 import {
   SessionReplayRemoteConfigFetch as AmplitudeSessionReplayRemoteConfigFetch,
+  SessionReplaySessionIDBStore as AmplitudeSessionReplaySessionIDBStore,
   SessionReplayConfig,
   SessionReplayRemoteConfig,
 } from './typings/session-replay';
 
-export const UNEXPECTED_NETWORK_ERROR_MESSAGE = 'Network error occurred, session replay remote config fetch failed';
-export const SUCCESS_REMOTE_CONFIG = 'Session replay remote config successfully fetched';
-export const MISSING_API_KEY_MESSAGE = 'Session replay remote config not fetched due to missing api key';
+const UNEXPECTED_NETWORK_ERROR_MESSAGE = 'Network error occurred, session replay remote config fetch failed';
+const SUCCESS_REMOTE_CONFIG = 'Session replay remote config successfully fetched';
+const SERVER_URL = '/sessions/v2/targeting.json';
```

Comment:
This will definitely change, this is just useful for local development until the API gets set up